src/backend/api/model/schedules.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+from .database import Base
+
+class Schedule(Base):
+    __tablename__ = 'schedules'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    schedule_id = Column(String(255), nullable=False)
+    title = Column(String(255), nullable=False)
+    description = Column(String(255), nullable=True)
+    start_time = Column(DateTime, nullable=False)
```

Comment:
Import 'datetime' module as it is used in the 'created_at' and 'updated_at' columns.